src/controllers/bundler.controller.ts

```diff
@@ -0,0 +1,43 @@
+import fs from 'fs';
+import path from 'path';
+import { NextFunction, Request, Response, Router } from 'express';
+import { ProblemException } from '../exceptions/problem.exception';
+import { Controller } from '../interfaces';
+import bundler from '@asyncapi/bundler';
+import { documentValidationMiddleware } from '../middlewares/document-validation.middleware';
+
+export class BundlerController implements Controller {
+  public basepath = '/bundle';
+  private async bundle(req: Request, res: Response, next: NextFunction) {
+    const asyncapis: Array<string> = req.body.asyncapis || [req.body.asyncapi];
+    const options: any = {};
```

Comment:
Please add type for that.

Reply:
Do we need options as parameters because there is the only base?